src/components/redux/books/booksSlice.js

```diff
@@ -0,0 +1,49 @@
+// import { createSlice } from '@reduxjs/toolkit';
+// import bookList from '../../home/bookLists';
+
+// const initialState = {
+//   bookList,
+// };
+
+// const bookSlice = createSlice({
+//   name: 'book',
+//   initialState,
+//   reducers: {
+//     addBook: (state, action) => {
+//       const newBook = action.payload;
+//       state.bookList.push(newBook);
+//     },
+//     removeBook: (state, action) => {
+//       const itemId = action.payload;
+//       state.bookList = state.bookList.filter((item) => item.id !== itemId);
+//     },
+//   },
+// });
+
+// export const { removeBook, addBook } = bookSlice.actions;
+// export default bookSlice.reducer;
```

Comment:
Kindly take away every commented piece of code in your codebase as it gives makes it messy and gives it a less professional look😉.